components/automate-cli/pkg/verifyserver/services/nfsmountservice/nfsmountservice.go

```diff
@@ -0,0 +1,260 @@
+package nfsmountservice
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+type INFSService interface {
+	GetNFSMountDetails(models.NFSMountRequest, bool) *[]models.NFSMountResponse
+}
+
+type NFSMountService struct {
+}
+
+func NewNFSMountService() INFSService {
+	return &NFSMountService{}
+}
+
+var mut *sync.Mutex = &sync.Mutex{}
+
+func makeConcurrentCall(ip string, test bool, node_type string, mountLocation string, ch chan string, nfsMountResultMap map[string][]models.NFSMountResponse, shareMap map[string]models.NFSMountLocResponse, key string, countMap map[models.NFSMountLocResponse]int) {
```

Comment:
No need of `test` parameter. You can run the mock server on a custom port. Please refer to the changes I have made to https://github.com/chef/automate/pull/7865